api/provider/aws/service_delete_test.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/api/tag"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func TestService_deleteService(t *testing.T) {
+
+	tagStore := tag.NewMemoryStore()
+	serviceProvider := NewServiceProvider(nil, tagStore, nil)
+	_ = serviceProvider
```

Comment:
?

Reply:
This was just there so compiler didn't throw an error, I'm wondering how testing should be setup. Looking for feedback here.

Reply:
Our current pattern in these test files is to write unit tests for the helper functions that don't make AWS calls; thus, we're not writing unit tests for the master `Delete()` function, nor the helper functions that reach out to AWS. As things stand right now, `Delete()` doesn't have any new helper functions that would need to be tested (both `scaleService()` and `deleteService()` call AWS).